<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>358</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Util for xwhatsit running QMK firmware</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="enterBootloaderPushbutton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>201</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter Bootloader</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="autoEnterModeCheckBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>411</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Check this before connecting your keyboard, if your keyboard is misbehaving, and sending lots of random characters.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Auto-enter bootloder when new device is found</string>
    </property>
   </widget>
   <widget class="QPushButton" name="keypressMinotorPushButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>201</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Keypress Monitor (simple)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>201</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Collect Test Data</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>290</y>
      <width>211</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Keypress Monitor (advanced)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="eraseEepromPushButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>260</y>
      <width>211</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Erase EEPROM, and reboot</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>320</y>
      <width>211</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Inspect Keymap</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>421</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>421</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>List of detected devices: (constantly refreshed)</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
